language.go

```diff
@@ -181,6 +181,7 @@ var Exts = map[string]string{
 	"f03":         "FORTRAN Modern",
 	"f08":         "FORTRAN Modern",
 	"gleam":       "Gleam",
+	"g4":          "ANTLR v4 Grammar",
```

Comment:
Can I ask you to change it to `ANTLR` so that it can be displayed regardless of version?

```suggestion
	"g4":          "ANTLR",
```

Reply:
The extension g4 is only for antlr v4.
g3 (not used anymore) is for antlr v3.
For me it's fine to change it. What is when somebody uses the old g3 Syntax?
Can "G3" also be Antlr and both are combined in the line count?

Reply:
I use now a shorter Name: ANTLR v4. I think we should keep v4.

Reply:
Yes, it makes sense. In fact, you can even assume that only v4 will be used in general.
v3 is already over 6 years old.